### DISCUSSION TITLE
Portamento glissando in Musescore 4

### DISCUSSION THREAD

**Discussion by EvkNet** [musescore/MuseScore#15110] [Q&A] [answered]
Why doesn't glissando portamento in strings work anymore in musescore 4?
Thanks in advance

**Comment by MarcSabatella** [comment, marked as answer]
It does for me, in MS Basic at least.  But I think it's just not implemented yet for Muse Sounds.  Odd, in that the samples are clearly there, since there is still some portamento whether you write it or not.  But I think the code to hook those samples up to the score marking is not present yet.

**Reply by EvkNet** [reply]
Thank for the reply
